serverlessworkflow/modules/ROOT/pages/cloud/quarkus/kubernetes-service-discovery.adoc

```diff
@@ -273,25 +273,45 @@ When using the Kubernetes service discovery feature, you need to balance if your
 
 If the URI pattern is not found in the application properties, then discovery is not triggered. However, the scanning is performed anyway. Therefore, a short time is required to go through the startup scan.
 
+=== Available service discovery implementations
+
+Currently, there are two implementations available.
+
 You can enable the Kubernetes service discovery by adding a service discovery implementation to the application's dependencies as shown in the following Maven dependencies:
 
+==== `org.kie.kogito:kogito-addons-quarkus-fabric8-kubernetes-service-catalog`
+
+When enabled, it uses the Kubernetes Java API to discover the services.
```

Comment:
Can you improve this description by saying that's ideally used without the operator in self-managed scenarios? (when users build and deploy workflows on their own).

Reply:
Later, I'll improve this by adding a link to the operator's guide (that we don't have it yet)

Reply:
Done.